src/exercises/exerciseTypes/Exercise.sc.js

```diff
@@ -35,7 +35,7 @@ const Exercise = styled.div`
   }
 
   .headline {
-    font-size: ;
+    font-size:;
```

Comment:
is this a thing?

Reply:
This is not a thing, not sure why this is here. But, I have removed it.

Reply:
Tak :)